cmd/oci-runtime-tool/generate.go

```diff
@@ -92,21 +91,28 @@ var generateCommand = cli.Command{
 	Before: before,
 	Action: func(context *cli.Context) error {
 		// Start from the default template.
-		specgen := generate.New()
+		var osPointer *string
```

Comment:
Maybe rename the variable to `specOs`? I don't like that we have `Pointer` in the name. Go doesn't recommend hungarian notation.